usaspending_api/etl/management/commands/elasticsearch_indexer.py

```diff
@@ -286,6 +344,27 @@ def set_config(passthrough_values: list, arg_parse_options: dict) -> dict:
             "unique_key_field": ES_COVID19_FABA_UNIQUE_KEY_FIELD,
             "write_alias": settings.ES_COVID19_FABA_WRITE_ALIAS,
         }
+    elif arg_parse_options["load_type"] == "recipient":
+        config = {
+            "base_table": "recipient_profile",
+            "base_table_id": "id",
+            "create_award_type_aliases": False,
+            "data_transform_func": transform_recipient_profile_data,
+            "data_type": "recipient",
+            "execute_sql_func": execute_sql_statement,
+            "extra_null_partition": False,
+            "field_for_es_id": "id",
+            "initial_datetime": default_datetime,
+            "max_query_size": settings.ES_RECIPIENTS_MAX_RESULT_WINDOW,
+            "optional_predicate": "",
+            "primary_key": "id",
+            "query_alias_prefix": settings.ES_RECIPIENTS_QUERY_ALIAS_PREFIX,
+            "required_index_name": settings.ES_RECIPIENTS_NAME_SUFFIX,
+            "sql_view": settings.ES_RECIPIENTS_ETL_VIEW_NAME,
+            "stored_date_key": ...,
```

Comment:
What does this `...` represent?

Reply:
`...` is usually used when omitting a value, but it's not the same as `None`. In this case `None` is probably the correct value to use since these `stored_date_key` isn't used for those indices.

If we use `...` and did a simple `if stored_date_key: <do something>` check then the `if` statement would pass since `...` isn't `None` or `False`.

For example:
```python
>>> x = ...

>>> type(x)
<class 'ellipsis'>

>>> y = None

>>> type(y)
<class 'NoneType'>

>>> if x: print("Value here")
Value here

>>> if y: print("Value here")
>>>
```

I can switch the `...` out for `None` in a commit to make them clearer.